src/RouteParamsExtractor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * This file is part of prooph/http-middleware.
+ * (c) 2016-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\HttpMiddleware;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * @author Florent Blaison
```

Comment:
git tracks you already, please remove this.

Reply:
@prolic Oh yes, that my IDE is doing that automaticaly 😄. I change that